api/apps/curations/model.coffee

```diff
@@ -20,12 +20,13 @@ OPTIONS = { allowUnknown: true, stripUnknown: false }
 #
 # Schemas
 #
-schema = (->
+@schema = (->
   id: @objectId()
   name: @string().allow('', null)
+  content: @any()
```

Comment:
This could be a solution for getting other fields in Curations, since it's a flexible data structure. We'd have to support `any` on the JoiQL side I believe. Happy to pull this part out until I sort things out with JoiQL!